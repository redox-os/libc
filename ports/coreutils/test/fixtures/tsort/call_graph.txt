main parse_options
main tail_file
main tail_forever
tail_file pretty_name
tail_file write_header
tail_file tail
tail_forever recheck
tail_forever pretty_name
tail_forever write_header
tail_forever dump_remainder
tail tail_lines
tail tail_bytes
tail_lines start_lines
tail_lines dump_remainder
tail_lines file_lines
tail_lines pipe_lines
tail_bytes xlseek
tail_bytes start_bytes
tail_bytes dump_remainder
tail_bytes pipe_bytes
file_lines dump_remainder
recheck pretty_name
