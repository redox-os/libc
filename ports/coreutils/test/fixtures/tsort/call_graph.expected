main
parse_options
tail_file
tail_forever
tail
recheck
write_header
tail_lines
tail_bytes
pretty_name
start_lines
file_lines
pipe_lines
xlseek
start_bytes
pipe_bytes
dump_remainder
